docs/spec/common-functions.md

```diff
@@ -18,6 +18,8 @@ let HashingOutput = Base64URL( Multihash(DATA, 'sha2-256') );
 
 ### Commitment Value Generation
 
-All commitment values created for the commit/reveal function of Sidetree operations ****MUST**** be sufficiently random values (e.g. a 32 byte cryptographically secure string) and ****MUST**** be unique across operations, without reuse in the operational lineage of a DID.
+Recovery commitment values ****MUST**** be the [JCS canonicalized](https://tools.ietf.org/html/draft-rundgren-json-canonicalization-scheme-17) [IETF RFC 7517](https://tools.ietf.org/html/rfc7517) JWK representation of the public key that is newly generated each new recovery commitment. Implementers and wallets ****MUST NOT**** reuse keypairs across recovery invocations.
```

Comment:
"Implementers and wallets ****MUST NOT**** reuse keypairs across recovery invocations." this assertion is not testable, probably should be a warning note and not normative text

Reply:
same with section below.